### PR TITLE
Add explanation for "Adding Children" in ColorPicker

### LINKED ISSUE #2406: Add a description of `children` in `ColorPicker`
### Subject

ColorPicker

### Description

Add a description of `children` in `ColorPicker`.

The description of `children` in `ColorPicker` does not exist on the documentation site.

https://yamada-ui.github.io/yamada-ui/?path=/story/components-forms-colorpicker--with-children

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by hironokobari**
Closes #2406

## Description

Add explanation for "Adding Children" in `ColorPicker`

## Is this a breaking change (Yes/No)

No

### CODE REVIEW

File: docs/contents/components/forms/color-picker/index.ja.mdx [hunk 1]
```diff
@@ -278,6 +278,22 @@ import { ColorPicker } from "@yamada-ui/react"
 <ColorPicker placeholder="#000000" withResult />
 ```
 
+### 子要素の追加
+
+`ColorPicker`コンポーネントに子要素を追加できます。
+
```
Comment (odendayoko) [inline docs/contents/components/forms/color-picker/index.ja.mdx:284]: @hironokobari
The content of the Japanese text and English text differs. 
The English version is more specific and clearer, so I recommend modifying the Japanese text to match the English version.

e.g.,「独自の要素を追加するには、ColorPickerコンポーネントに子要素を挿入してください。」

File: docs/contents/components/forms/color-picker/index.mdx [hunk 2]
```diff
@@ -274,6 +274,22 @@ To make read-only, set `isReadOnly` to `true`.
 <ColorPicker placeholder="#000000" withResult />
 ```
 
+### Adding Children
```
Comment (odendayoko) [inline docs/contents/components/forms/color-picker/index.mdx:277]: @hironokobari
Most headings use verbs in the imperative form (e.g., "Change Variant", "Set Default Value"). 
To maintain consistency with other headings in the document, could you change the heading to `Add Children`?

**Review by odendayoko — CHANGES_REQUESTED** [review]
@hironokobari
Thank you for the implementation.
 I apologize for the minor feedback, but I have left some comments. Could you please make the necessary adjustments?

File: docs/contents/components/forms/color-picker/index.mdx (hunk 2 above)
Comment (odendayoko) [inline docs/contents/components/forms/color-picker/index.mdx:277]: @hironokobari
Most headings use verbs in the imperative form (e.g., "Change Variant", "Set Default Value").
To maintain consistency with other headings in the document, could you change the heading to Add Children?

File: docs/contents/components/forms/color-picker/index.ja.mdx [hunk 3]
```diff
@@ -278,6 +278,22 @@ import { ColorPicker } from "@yamada-ui/react"
 <ColorPicker placeholder="#000000" withResult />
 ```
 
+### 子要素の追加
+
+`ColorPicker`コンポーネントに子要素を追加できます。
```
Comment (odendayoko) [inline docs/contents/components/forms/color-picker/index.ja.mdx:283]: @hironokobari
The content of the Japanese text and English text differs.
The English version is more specific and clearer, so I recommend modifying the Japanese text to match the English version.

e.g.,「独自の要素を追加するには、ColorPickerコンポーネントに子要素を挿入してください。」

**Review by odendayoko — CHANGES_REQUESTED** [review]
@hironokobari
Thank you for the implementation.
I apologize for the minor feedback, but I have left some comments. Could you please make the necessary adjustments?

**Review by odendayoko — APPROVED** [review]
@hironokobari
Thank you! LGTM

File: docs/contents/components/forms/color-picker/index.ja.mdx [hunk 4]
```diff
@@ -278,6 +278,22 @@ import { ColorPicker } from "@yamada-ui/react"
 <ColorPicker placeholder="#000000" withResult />
 ```
 
+### 子要素の追加
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/color-picker/index.ja.mdx:281]: Change `子要素の追加` to `ドロップダウンに要素を追加する`.

File: docs/contents/components/forms/color-picker/index.ja.mdx [hunk 5]
```diff
@@ -278,6 +278,22 @@ import { ColorPicker } from "@yamada-ui/react"
...
+
+独自の要素を追加するには、`ColorPicker`コンポーネントに子要素を挿入してください。
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/color-picker/index.ja.mdx:283]: Change to

```
独自の要素を追加するには、`ColorPicker`コンポーネントに子要素を挿入してください。
```

```
ドロップダウンに要素を追加する場合は、`children`に要素を設定します。
`children`には、`value`と`onClose`が提供されます。これを利用して、内部状態にアクセスすることができます。
```

File: docs/contents/components/forms/color-picker/index.mdx [hunk 6]
```diff
@@ -274,6 +274,22 @@ To make read-only, set `isReadOnly` to `true`.
 <ColorPicker placeholder="#000000" withResult />
 ```
 
+### Add Children
+
+To add custom content, insert children elements into the `ColorPicker` component.
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/color-picker/index.mdx:279]: Change

```
To add custom content, insert children elements into the `ColorPicker` component.
```

to

```
To add elements to a dropdown, set the elements to the `children` property.
The `children` are provided with `value` and `onClose` methods, which can be used to access the internal state.
```

File: docs/contents/components/forms/color-picker/index.mdx [hunk 7]
```diff
@@ -274,6 +274,22 @@ To make read-only, set `isReadOnly` to `true`.
 <ColorPicker placeholder="#000000" withResult />
 ```
 
+### Add Children
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/color-picker/index.mdx:277]: Change `Add Children` to `Adding elements to a dropdown`.

**Comment by hironokobari** [comment]
@hirotomoyamada @odendayoko 
Thanks for pointing it out! Fixed.

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**